test/postman_test/BCDA_Tests_Sequential.postman_collection.json

```diff
@@ -46,8 +50,8 @@
 					"script": {
 						"id": "00a9ceb7-a32e-4a21-a689-201af70162bf",
 						"exec": [
-							"pm.test(\"Response time is less than 200 ms\", function() {",
-							"    pm.expect(pm.response.responseTime).to.be.below(200);",
+							"pm.test(\"Response time is less than 100 ms\", function() {",
+							"    pm.expect(pm.response.responseTime).to.be.below(100);",
```

Comment:
Will this really get reverted in the merge?

Reply:
Yikes!  I wonder why the automatic merge took my version of the file over the version coming from `master`!   Fixed.